README.md

```diff
@@ -27,11 +27,21 @@ beautiful Material Design and
 Cupertino (iOS-flavor) widgets, rich motion APIs,
 smooth natural scrolling, and platform awareness.
 
+<a href="https://github.com/flutter/flutter/tree/master/examples/flutter_gallery/lib/demo/animation">
```

Comment:
Can we use markdown formatting for these links, please?

Reply:
@mit-mit  by using markdown formatting, we'll have to sacrifice image sizing

Reply:
`[![alt](https://flutter.io/images/homepage/screenshot-2.png)](https://github.com/flutter/posse_gallery)`

meaning we can't add image size to markdown formatting

Reply:
@mit-mit  DONE SIR! 👍